Codeinwp/neve
# feat: new options for mobile sidebar

### Summary

**DEPENDS ON https://github.com/Codeinwp/neve/pull/1848**. Should be merged after that one.

Adds new options for the Sidebar Row: 
- new open behaviors
- new sidebar width control
- tested with AMP


### Will affect visual aspect of the product

NO

### Test instructions


- Check that the new open behaviors work fine.
- Check that everything works with AMP.


Closes #1717.
